README.md

```diff
@@ -1,2 +1,19 @@
-# yad2_bouncer
-A script for bouncing yad2 ads
+# Yad2 Bouncer
+A script for bouncing yad2 ads.
+
+# Usage
```

Comment:
Add a section or at least mention the a support for a GUI-less Linux environment.